tests/sentry/notifications/utils/test_get_scope.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase
+
+from sentry.models import Organization, Project, Team, User
+from sentry.notifications.helpers import get_scope
+from sentry.notifications.types import NotificationScopeType
+
+
+class GetScopeTestCase(TestCase):
+    def setUp(self) -> None:
+        self.user = User(id=1)
```

Comment:
I don't think I've seen tests written like this before - why not use `self.create_user`?

Reply:
I'm importing `TestCase` with `from unittest import TestCase` instead of `from sentry.testutils import TestCase`.
This saves a ton of overhead in these unit tests because I don't care about state at all.

Before:
```== 4 passed in 7.96s ==```
After:
```== 4 passed in 0.19s ==```